samples/bluetooth/tmap_peripheral/src/main.c

```diff
@@ -12,6 +12,7 @@
 #include <zephyr/sys/byteorder.h>
 
 #include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/byteorder.h>
```

Comment:
Hmmm... @jhedberg @Thalley we have an explicit `byteorder.h` under Bluetooth? How different is it from `<zephyr/sys/byteorder.h>` ?

Reply:
nevermind.